README.md

```diff
@@ -171,3 +172,70 @@ Set the MQTT username and password client settings.
 
 ##### Token Authentication
 Set the MQTT password to the token body, currently username will be disregarded but MUST be set to some value as this is required by the MQTT specification.
+
+
+### Enabling TLS
+
+MoP currently supports TLS transport encryption.
+
+#### TLS with broker
+
+1. Config mqtt broker to load tls config.
+```java
+MQTTServerConfiguration mqtt = new MQTTServerConfiguration();
+mqtt.setTlsEnabled(true);
+mqtt.setTlsCertificateFilePath(TLS_SERVER_CERT_FILE_PATH);
+mqtt.setTlsKeyFilePath(TLS_SERVER_KEY_FILE_PATH);
```

Comment:
We should provide how to configure with the configuration file?

Reply:
Right, I will update to doc.